modules/installation-configuration-parameters.adoc

```diff
@@ -299,9 +305,40 @@ ifdef::osp[]
 |`platform.openstack.lbFloatingIP`
 |An existing floating IP address to associate with the load balancer API.
 |An IP address, for example `128.0.0.1`.
+|====
+
+.Optional {rh-openstack} parameters
+[%header, cols=".^2,.^3,.^5a"]
+|====
+|Parameter|Description|Values
+
+|`compute.platform.openstack
```

Comment:
I'm not wild about the extra hard return. It makes it look like it's rendering like the other tables, but it isn't. The only difference I see between your table and mine is that I have a variable in angled brackets.

Reply:
Agreed. Reverting that.

Reply:
Created #23257